src/containers/Login/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component, FormEvent } from 'react';
+import { FormGroup , ControlLabel, FormControl, Button, FormControlProps } from 'react-bootstrap';
+import './Login.css';
+
+import { Auth } from "aws-amplify";
+
+interface ILoginProps {
+  email: '',
+  password: ''
+}
+
+interface ILoginFormState {
+  email: string,
+  password: string 
+}
+
+
+class Login extends Component<ILoginProps, ILoginFormState> {
```

Comment:
if possible, try to use the new functional component style instead of class, so we can take advantage of react suspense.

Reply:
Yup, will do that. thanks